run_scripts/run_pipeline.sh

```diff
@@ -40,5 +40,6 @@ echo "Starting run with id '$RUN_ID'"
 
 ./6_backup_data_root.sh "$DATA_ROOT" "$DATA_BACKUPS_DIR/data-$RUN_ID.tar.gzip"
 
-./7_upload_files.sh "$USER" "$AVF_BUCKET_CREDENTIALS_PATH" "$PIPELINE_CONFIGURATION" "$RUN_ID" "$DATA_ROOT" \
-    "$PERFORMANCE_LOGS_DIR/memory-$RUN_ID.profile" "$DATA_BACKUPS_DIR/data-$RUN_ID.tar.gzip"
+./7_upload_analysis_files.sh "$USER" "$AVF_BUCKET_CREDENTIALS_PATH" "$PIPELINE_CONFIGURATION" "$RUN_ID" "$DATA_ROOT"
+
+./8_upload_log_files.sh "$USER" "$AVF_BUCKET_CREDENTIALS_PATH" "$PIPELINE_CONFIGURATION" "$PERFORMANCE_LOGS_DIR" "$DATA_BACKUPS_DIR"
```

Comment:
I agree that it's preferable to run this as part of the pipeline if we can. Only thing to note here is that the sequencing of the steps here means if the analysis_files repeatedly fail to upload then you'll start consuming lots of disk space with archives that aren't removed until we notice there's a problem. I think this is what we want though - we should be monitoring often enough to notice before we run out of space, and I think it's safer to have archives for partially-uploaded data than none at all. Just making sure you've thought about this and agree :)

Reply:
Discussed offline - to keep as is